== TEST (2019-06-25 11:26:26) 
-- CODE:
from numpy import*
from numpy.linalg import*
saida = array([50, -120, 350, 870])
saida = saida.t
eq = array([[1,1,0,0],[0,1,1,0]. [0,0,1,0].[1,0,0,1]])

saida = dot(inv(eq), saida)
print(saida)
-- ERROR:
File "XXXX", line 5
    eq = array([[1,1,0,0],[0,1,1,0]. [0,0,1,0].[1,0,0,1]])
                                     ^
SyntaxError: invalid syntax
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-25 11:26:51) 
-- CODE:
from numpy import*
from numpy.linalg import*
saida = array([50, -120, 350, 870])
saida = saida.t
eq = array([[1,1,0,0],[0,1,1,0], [0,0,1,0],[1,0,0,1]])

saida = dot(inv(eq), saida)
print(saida)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-25 11:26:59) 
-- CODE:
from numpy import*
from numpy.linalg import*
saida = array([50, -120, 350, 870])
saida = saida.T
eq = array([[1,1,0,0],[0,1,1,0], [0,0,1,0],[1,0,0,1]])

saida = dot(inv(eq), saida)
print(saida)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-25 11:27:07) 
-- CODE:
from numpy import*
from numpy.linalg import*
saida = array([50, -120, 350, 870])
saida = saida.T
eq = array([[1,1,0,0],[0,1,1,0], [0,0,1,0],[1,0,0,1]])

saida = dot(inv(eq), saida)
print(saida)
-- EXECUTION TIME:
1.30359
-- TEST CASE 1:
---- input:
Não_há
---- correct output:
[280. 230. 350. 590.]
---- user output:
[ 520. -470.  350.  350.]
-- GRADE:
0%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-25 11:27:19) 
-- CODE:
from numpy import*
from numpy.linalg import*
saida = array([50, -120, 350, 870])
saida = saida.T
eq = array([[1,1,0,0],[0,1,1,0], [0,0,1,0],[1,0,0,1]])

saida = dot(inv(eq), saida)
print(saida)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== TEST (2019-06-25 11:28:15) 
-- CODE:
from numpy import*
from numpy.linalg import*
saida = array([50, -120, 350, 870])
saida = saida.T
eq = array([[1,-1,0,0],[0,1,-1,0], [0,0,1,0],[1,0,0,1]])

saida = dot(inv(eq), saida)
print(saida)
-- ERROR:
Traceback (most recent call last):
  File "XXXX", line 1, in <module>
    from numpy import*
ModuleNotFoundError: No module named 'numpy'
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-06-25 11:28:18) 
-- CODE:
from numpy import*
from numpy.linalg import*
saida = array([50, -120, 350, 870])
saida = saida.T
eq = array([[1,-1,0,0],[0,1,-1,0], [0,0,1,0],[1,0,0,1]])

saida = dot(inv(eq), saida)
print(saida)
-- EXECUTION TIME:
1.17617
-- TEST CASE 1:
---- input:
Não_há
---- correct output:
[280. 230. 350. 590.]
---- user output:
[280. 230. 350. 590.]
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
